app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatabaseService.kt

```diff
@@ -96,12 +96,14 @@ class FirestoreDatabaseService internal constructor(
         lateinit var self: DocumentReference
         var displayName: String? = null
         var email: String? = null
+        var userPrivilege : String? = null
```

Comment:
I don't think this belongs in this pr

Reply:
yep

Reply:
Yes , I will resolve that with the requested changes coming